src/test/java/plugins/DashboardViewPluginTest.java

```diff
@@ -197,126 +192,6 @@ public void latestsBuildsPortlet_onlyLatest() {
         assertThat(latestBuilds.hasBuild(1), is(false));
     }
 
-    @Test
-    public void testStatisticsChart_success() throws IOException {
-        DashboardView v = createDashboardView();
-        TestStatisticsChartPortlet chart = v.addBottomPortlet(TestStatisticsChartPortlet.class);
-        v.save();
-
-        FreeStyleJob successJob = createFreeStyleJob(job -> {
-            String resultFileName = "status.xml";
-            job.addShellStep(
-                "echo '<testsuite><testcase classname=\"\">" +
-                    "</testcase></testsuite>'>" + resultFileName
-            );
-            job.addPublisher(JUnitPublisher.class).testResults.set(resultFileName);
-        });
-
-        buildSuccessfulJob(successJob);
-
-        v.open();
-
-        Map<Integer, Integer> colorsOccurrences = getColorDistributionOfSignificantColors(chart.getImage());
-
-        assertThat(colorsOccurrences.size(), is(1));
-
-        Integer color = colorsOccurrences.keySet().iterator().next();
-        int blueComp = (color << 24) >>> 24;
-        int greenComp = (color << 16) >>> 24;
-        int redComp = (color << 8) >>> 24;
-
-        assertThat(blueComp, greaterThan(150));
```

Comment:
the colour changed in core, I don't think this is a great test for here to have.

Too fragile to change.

I've left the tests in place that don't look at colours in an image